profiles/commands.py

```diff
@@ -15,6 +20,38 @@ def _update_name_from_orcid_record(profile: Profile, orcid_record: dict) -> None
                             '{}, {}'.format(family_name, given_name))
 
 
+def _update_affiliations_from_orcid_record(profile: Profile, orcid_record: dict) -> None:
+    orcid_affiliations = orcid_record.get('activities-summary', {}).get('employments', {}).get(
+        'employment-summary', {})
+
+    for index, orcid_affiliation in enumerate(orcid_affiliations):
+        organization = orcid_affiliation['organization']
+        address = organization['address']
+
+        affiliation = Affiliation(str(orcid_affiliation['put-code']),
+                                  countries.get(address['country']), organization['name'],
+                                  date(**{k: int(v) for k, v in
+                                          orcid_affiliation['start-date'].items()}),
+                                  orcid_affiliation.get('department-name'), address.get('city'),
+                                  address.get('region'),
+                                  date(**{k: int(v) for k, v in
```

Comment:
conceptually if this is a date, it has no hour/minutes/seconds. I guess there is no correct data type to store `2017-01-01`?